vendor/manifest

```diff
@@ -226,6 +226,15 @@
 			"notests": true
 		},
 		{
+			"importpath": "golang.org/x/net/context",
+			"repository": "https://go.googlesource.com/net",
+			"vcs": "git",
+			"revision": "a6577fac2d73be281a500b310739095313165611",
+			"branch": "master",
```

Comment:
Can we use dedicated branch for go1.7 `https://go.googlesource.com/net/+/release-branch.go1.7/context/`